src/pages/Ranking.tsx

```diff
@@ -29,7 +29,7 @@ const Ranking = () => {
   const place = serachParams.get('place') || '서울';
```

Comment:
```suggestion
  const place = searchParams.get('place') || '서울';
```

오타... 
`serach`가 아니라 `search`가 올바른 단어같아요